activemodel/lib/active_model/validations/callbacks.rb

```diff
@@ -23,6 +23,7 @@ module Callbacks
       included do
         include ActiveSupport::Callbacks
         define_callbacks :validation,
+                         terminator: deprecated_false_terminator(ActiveModel::Validations),
```

Comment:
When it comes to deprecation warnings I think it would be nicer to have it as static as possible (easier grepping perhaps).

So I'm :+1: on saying `returning false in Active Record and Active Model callbacks in deprecated` (remember the spaces in the names when we aren't referring to the constants :grin:)

@claudiob I still don't understand why Active Support callbacks aren't affected by this — can you elaborate? I especially don't understand given this: https://github.com/rails/rails/pull/21218/files#r39675510